### DISCUSSION TITLE
What does bdelete and bwipeout do?

### DISCUSSION THREAD

**Discussion by dominik-lukacs** [LunarVim/LunarVim#4036] [Q&A]
I'm just setting up lvim and was searching for a way to close buffers and found these two ways. However, I am not sure what the difference is or if these are the intended way to close a buffer in a keymap

**Comment by LostInTheLogs** [comment]
`:h bdelete` `:h bwipeout`
 
 Use `:BufferKill` (our command) if you want to preserve the layout
